sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -368,6 +389,16 @@ export interface PolicySubjectProperties {
     subjectAlternativeNames: SubjectAlternativeNames;
 }
 
+// @public
+export interface PollerLikeWithPublicState<TPublic, TState, TResult> extends PollerLike<TState, TResult> {
```

Comment:
`PollerWithProperties` ?

Reply:
Sure, but let's agree on something here first: https://github.com/Azure/azure-sdk-for-js/pull/6621#discussion_r359583391

Reply:
Since this is being exported from keyvault-certificates does it make more sense to make this more specific rather than  the generic version which we expect to be in core-lro?